CHANGES.md

```diff
@@ -72,6 +72,8 @@ should handle this. ([#25252](https://github.com/apache/beam/issues/25252)).
 
 ## Breaking Changes
 
+* `upload_graph` is not needed since the graph now is uploaded automatically when it is larger than 10MB for Java SDK ([PR#28621](https://github.com/apache/beam/pull/28621).
```

Comment:
This does not sound like a `breaking change` instead of an improvement, i.e., in which scenario this would break existing user?

If so, consider phrasing like: `upload_graph` experiments option is no longer required in the case of large pipeline graphs ...

Reply:
Done. Thanks.